### ISSUE TITLE
Audit Admin Access

### ISSUE THREAD

**Issue by DanCoughlin** [psu-libraries/scholarsphere-3#1477]
Go through the UMG that has access to scholarsphere.psu.edu/admin and make sure everyone that has access still should. We should set up a reminder to do this every 6 or 12 months.

**Comment by carolyncole** [comment]
@dmc186 Unfortunately I do not have privileges to remove people once they have been added.  There is an issue with the UI for UMgs.  For example Jeff Manelli asked to be removed, but that was not possible.

**Comment by carolyncole** [comment]
There are 2 groups to recreate: umg/up.dlt.scholarsphere-admin, umg/up.dlt.scholarsphere-admin-viewers.  @DanCoughlin  will create the new group and look at the existing groups.  The config files will then need to be updated.

**Comment by DanCoughlin** [comment]
Reviewing scholarsphere-admin and scholarsphere-admin-viewers it appears that once we delete users that shouldn't be there, then the overlap is pretty significant. I wonder if we want 2 new groups still? I've created a new group, up.libraries.scholarsphere.service-team, I did this thinking a new name would be much easier for us to identify the group instead of only changing the name slightly. If this is misguided, I'll change the group names to something more similar. Group names also sync to AD and Outlook. If I removed the proper people, then the overlap is that Karen is in one group while everyone else is in both groups. For now, I've created one group:

- up.libraries.scholarsphere.service-team is to replace up.dlt.scholarsphere-admin-viewers. I removed Rob, Bess Sadler, and Mahir Akgun from this group thinking they shouldn't be there. Also @cam156 is an owner of this group (Adam, Mike, Justin are admins) so she should be able to do anything I've done. Let me know if any edits should be made.

If this is ok, then we need another ticket to delete the old groups once configuration changes are made.

**Comment by carolyncole** [comment]
We will only keep one group since the groups overlap and we believe there is no harm that can be done with access to resque.

**Comment by carolyncole** [comment]
Here are all the places that need to be changed in the code
```
cam156@L5-PW3-153-233[~/scholarsphere]> bgrep scholarsphere-admin
app/models/user.rb
82:    groups.include? 'umg/up.dlt.scholarsphere-admin-viewers'

config/initializers/resque_admin.rb
8:    current_user.groups.include? 'umg/up.dlt.scholarsphere-admin'

config/initializers/stats_admin.rb
8:    current_user.groups.include? 'umg/up.dlt.scholarsphere-admin-viewers'

spec/factories/users.rb
42:      group_list 'umg/up.dlt.scholarsphere-admin-viewers'
81:        group_list_array = ['umg/up.dlt.scholarsphere-admin.admin1', 'umg/up.dlt.scholarsphere-admin.admin2']
89:        group_list_array = (0..100).map { |i| "umg/up.dlt.scholarsphere-admin.admin#{i}" }

spec/controllers/downloads_controller_spec.rb
48:        allow_any_instance_of(User).to receive(:groups).and_return(['umg/up.dlt.scholarsphere-admin-viewers'])

```

**Comment by awead** [comment]
Fixed in #1545
